ui/package.json

```diff
@@ -6,11 +6,12 @@
   "license": "Apache-2.0",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "pnpm generate:routes && tsc && vite build",
```

Comment:
Could we do the same change for `codegen`?

Reply:
We could, but we need the Core running for that to get the OpenAPI JSON. If it runs and returns the spec without any of the services (db, keycloak), then it could be relatively simple.